apps/web/src/components/example.tsx

```diff
@@ -22,14 +30,22 @@ export const Example: React.FC<Readonly<ExampleProps>> = ({
   ...props
 }) => {
   const emailName = path.split("/").pop();
-  if (path.length === 0 || !emailName) {
+  if (!path || !emailName) {
     throw new Error("Cannot have an empty path for an Example!");
   }
 
+  const [imageSrc, setImageSrc] = React.useState<string>(
```

Comment:
```suggestion
  const [imageSrc, setImageSrc] = React.useState(
```

I believe for this case TS will infer the type correctly since we give a default value, no need to pass the generic